eng/build/Workers.Powershell.props

```diff
@@ -8,8 +8,9 @@
 
   <Target Name="RemovePowershellWorkerRuntimes" BeforeTargets="AssignTargetPaths" Condition="'$(RuntimeIdentifier)' != ''">
     <ItemGroup>
-      <_KeepPowerShellRuntime Include="win;win-x86;win10-x86;win-x64;win10-x64" Condition="$(RuntimeIdentifier.StartsWith(win))" />
-      <_KeepPowerShellRuntime Include="linux;linux-x64;unix" Condition="$(RuntimeIdentifier.StartsWith(linux))" />
+      <_KeepPowerShellRuntime Include="win;win-x86;win10-x86;win-x64;win10-x64;win-arm64" Condition="$(RuntimeIdentifier.StartsWith(win))" />
+      <_KeepPowerShellRuntime Include="linux;linux-x64;linux-arm64;unix" Condition="$(RuntimeIdentifier.StartsWith(linux))" />
```

Comment:
Same for linux and osx - we can be specific about linux-x64 vs linux-arm64 and osx-x64 vs osx-arm64